ament_cmake_uncrustify/cmake/ament_uncrustify.cmake

```diff
@@ -65,6 +67,17 @@ function(ament_uncrustify)
     list(APPEND cmd "--exclude" "${ARG_EXCLUDE}")
   endif()
   list(APPEND cmd ${ARG_UNPARSED_ARGUMENTS})
+  if(NOT ARG_TIMEOUT)
+    if(WIN32)
```

Comment:
I think it is worth putting a comment here as to why we are increasing the Windows timeout.